Dockerfile

```diff
@@ -28,7 +28,8 @@ ENV ENABLE_WEBHOOKS=${ENABLE_WEBHOOKS}
 
 # Use ubi-micro as minimal base image to package the manager binary
 # See https://catalog.redhat.com/software/containers/ubi9/ubi-micro/615bdf943f6014fa45ae1b58
-FROM registry.access.redhat.com/ubi9/ubi-micro:9.1.0
+# use 9.2 over 9.1.0 due to CVEs (glibc-common-2.34-40.el9_1.1 and libcap-2.48-8.el9)
```

Comment:
Thanks for updating and I definitely agree. But do we need this comment? Ideally, updating image references should be done regularly anyway. I don't think we need to list what bugfixes were included in the currently used version.

Reply:
I guess we can remove. we can see the fixes via the linked JIRA issue